packages/react/src/tabs/indicator/useTabsIndicator.ts

```diff
@@ -124,6 +136,11 @@ export interface ActiveTabPosition {
   bottom: number;
 }
 
+export interface ActiveTabSize {
```

Comment:
I was in doubt if "dimensions" was a better name than "size". If you have a preferred termn, let me know.

Reply:
Size is ok (it's shorter and equally descriptive imo).